voltron.txt

```diff
@@ -29435,6 +29563,7 @@ dimwishlist:item=1506719573&perks=1467527085,3230963543,205890336,1546637391
 dimwishlist:item=1506719573&perks=3250034553,3230963543,205890336,1546637391
 
 // Trophy Hunter (curated / M+KB)
+//notes: PvP pick
```

Comment:
There were issues.. need to fix this.. Lemme see about doing a quick update.. This is an issue